docker/crabserver/addGH.sh

```diff
@@ -13,9 +13,16 @@ CRABServerDir=`realpath /data/srv/current/sw*/*/cms/crabserver/*`
 # 1. find which GitHub tags were installed
 # beware that directory name in $CRABServerDir may not be the GH tag but rather
 # name assigned at build time like v3.230303-comp4
-export PYTHONPATH=${CRABServerDir}/lib/python2.7/site-packages
-CRABServerGHTag=`python -c "from CRABInterface import __version__; print __version__"`
-WMCoreGHTag=`python -c "from WMCore import __version__; print __version__"`
+source `realpath /data/srv/HG*/apps/crabserver/etc/profile.d/init.sh`
+if type python3; then 
```

Comment:
This will not give you what you want, e.g.
```
type bla
bla not found

type python3
python3 is /opt/local/bin/python3
```
Much better is to use `command` bash tool to check for existence of some program, see https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script#677212

Reply:
As per Stefano suggestion, this is not needed anymore.

In any case, it relied on `python3` not being present in our py2 docker image, so that `type python3` returned non-zero exit code. 

Thanks for pointing `command -v`, always better to respect POSIX when we can, i will use it next time i need it!